### PR TITLE
LNP-1013: add sentry

### PR DESCRIPTION
**Pull request by madebyzak**
This PR;
- Adds Sentry in preparation for better error logging and the introduction of Session Replay. 

https://ministryofjustice.sentry.io/issues/?project=4508126598594560&referrer=sidebar&statsPeriod=14d
https://dsdmoj.atlassian.net/browse/LNP-1013

### CODE REVIEW

File: helm_deploy/values-prod.yaml [hunk 1]
```diff
@@ -16,6 +16,7 @@ generic-service:
     ADJUDICATIONS_API_URL: 'https://manage-adjudications-api.hmpps.service.justice.gov.uk'
     LAUNCHPAD_AUTH_URL: 'https://launchpad-auth.hmpps.service.justice.gov.uk'
     GA4_SITE_ID: 'G-7M5BW140G4'
+    SENTRY_DSN: 'https://f36188d0543b13fa07be85212df8e953@o345774.ingest.us.sentry.io/4508126598594560'
```
Comment (psoleckimoj) [inline helm_deploy/values-prod.yaml:19]: I think this is in the k8s secrets?

Comment (psoleckimoj) [reply helm_deploy/values-prod.yaml:19]: Or rather it is in content hub, maybe do the same?

Comment (madebyzak) [reply helm_deploy/values-prod.yaml]: Added to k8s secrets 👍

**Review by psoleckimoj — DISMISSED** [review]
LGTM!

**Review by psoleckimoj — APPROVED** [review]
LGTM!

**Pull request merged**